src/D2L.Bmx/PrintHandler.cs

```diff
@@ -25,10 +28,24 @@ AwsCredsCreator awsCredsCreator
 		} else if( string.Equals( output, "json", StringComparison.OrdinalIgnoreCase ) ) {
 			PrintJson( awsCreds );
 		} else {
-			if( OperatingSystem.IsWindows() ) {
-				PrintPowershell( awsCreds );
-			} else if( OperatingSystem.IsMacOS() || OperatingSystem.IsLinux() ) {
-				PrintBash( awsCreds );
+			var current = Process.GetCurrentProcess();
+			var parent = ProcessCodeMethods.GetParentProcess( new PSObject( current ) ) as Process;
+			string? parentProcName = parent?.ProcessName;
+			switch( parentProcName ) {
+				case "pwsh":
+					PrintPowershell( awsCreds );
+					break;
+				case "bash":
+				case "zsh":
```

Comment:
This is terminal on mac. Was debating on leaving a comment

Reply:
Don't think that is necessary, its a common enough shell even with Linux users which we want to support